stack.yaml

```diff
@@ -48,12 +52,9 @@ extra-deps:
 - hex-text-0.1.0.0
 - constraints-0.11@sha256:44f80a313d5d4fcfb3c18eb6625dd3db4bb845d5ca8d3b1f62125a4b7241e590
 - pretty-terminal-0.1.0.0@sha256:e9135d86ebb2a8e3aaf5a79088de4628dbd49988388e0fbfc26c5ecb3c399ad9
```

Comment:
For each package in this list please check whether it's still needed here.
* `aeson-options` is deprecated, it'd be nice to stop using it.
* `o-clock-1.1.0` is available in `lts-16.5`.

Maybe something else is not necessary.